minicluster/src/main/java/alluxio/multi/process/MultiProcessCluster.java

```diff
@@ -184,7 +189,9 @@ public synchronized void start() throws Exception {
     mProperties.put(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS,
         PathUtils.concatPath(mWorkDir, "underFSStorage"));
     new File(mProperties.get(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS)).mkdirs();
-    formatJournal();
+    if (!mNoFormat) {
```

Comment:
Could we avoid the double negative?

Reply:
I prefer to follow the convention of having the default be `false` for boolean options.